src/index.js

```diff
@@ -667,9 +665,10 @@ process.nextTick(run);
 // Global variable assignments
 // This section is broken out because it will eventually be deprecated
 function setGlobals() {
```

Comment:
Given the way things are done, this could probably be f.setGlobals = function() {.

Reply:
Yes, that would be the better way , i will make the correction.